certified-connectors/DocuSign/apiDefinition.swagger.json

```diff
@@ -21,7 +21,7 @@
   "paths": {
     "/accounts/{accountId}/connect": {
       "post": {
-        "summary": "When an envelope status changes (Connect)",
+        "summary": "DEPRECATED: When an envelope status changes (Connect)",
```

Comment:
@sriharip-docusign If you are deprecating the operation, We would recommend you to set the "deprecated" property to "true" and test the connector before pushing the changes to your PR.

Reply:
@sriyen-msft , I want it to be a 2 stage deprecation. Setting the "deprecated" property to "true", would mean that the trigger or action will not show up in the list. Some time in next few months, we will set the deprecated property to true and then actually deprecate it.